int16 TimeStep
string Agent
float64[] Bias
string Target
float64[] FullMu
int16[] FullMuDim
float64[] FullCov
int16[] FullCovDim
float64[] TMu
int16[] TMuDim
float64[] SMu
int16[] SMuDim
float64[] TCov
int16[] TCovDim
float64[] SCov
int16[] SCovDim